README.md

```diff
@@ -17,38 +17,48 @@ Rovercode runs on a [Raspberry Pi 3](https://www.raspberrypi.org/products/raspbe
 ## Setup
 
 ### Creating Your .env
-First, create a rovercode.com account [here](https://app.rovercode.com/accounts/login). Then, navigate to the "My Rovers" section and
-create a new rover. Once it is created, click the "Download Credentials" button at the bottom of the rover's detail page. The file
-will download as something like `rovercode_yourrovername.env`. Rename the file as only `.env` (nothing before the dot) and save it in the same directory as this README.
+First, create an app.rovercode.com account [here](https://app.rovercode.com/accounts/login).
+Then, navigate to the "My Rovers" section and create a new rover.
+Once it is created, click the "Download Credentials" button at the bottom of the rover's detail page.
+The file will download as something like `rovercode_yourrovername.env`.
+Save this file to a flash drive.
 
 ### Rover Setup
```

Comment:
When installing on the Rover, we only need to install Docker, then put the systemd service files and the scripts in the right places.